package.json

```diff
@@ -96,6 +96,11 @@
           "type": "string",
           "default": "",
           "description": "A root message for dotnet test command. In English it is 'The following Tests are available:' but for example in Polish it is 'Dostępne są następujące testy:'"
+        },
+        "dotnet-test-explorer.useTreeView": {
+          "type": "boolean",
+          "default": false,
```

Comment:
I tested the new code with MSTest. Didn't realize the output would be so different. 

Because MSTest does not output the full namespaces you cannot get a tree view. Also, if there are multiple tests with the same name they would show up multiple times in the test list but now get collapsed into the same item in the tree view. I personally don't think this is a problem because even when all tests with the exact same were listed, if you right clicked any one of them to run the test, all would be executed.